src/react/src/__tests__/utils.tests.js

```diff
@@ -2575,4 +2576,81 @@ describe('slcValidationSchema', () => {
             'Parent company cannot exceed 200 characters.'
         );
     });
-});
\ No newline at end of file
+});
+
+describe('formatExtendedField', () => {
+    const baseProps = {
+        created_at: '2025-10-01T12:00:00Z',
+        contributor_name: 'Test Contributor',
+        is_verified: true,
+        is_from_claim: false,
+    };
+
+    it('formats value with default formatter and fallback key', () => {
+        const result = formatExtendedField({
+            ...baseProps,
+            value: ['Alpha', 'Beta'],
+            field_name: 'native_language_name',
+        });
+
+        expect(result.secondary).toBe('October 1, 2025 by Test Contributor');
+        expect(result.embeddedSecondary).toBe('October 1, 2025');
+        expect(result.isVerified).toBe(true);
+        expect(result.isFromClaim).toBe(false);
+        expect(result.primary).toHaveLength(2);
+        expect(result.primary[0].props.children).toBe('Alpha');
+        expect(result.primary[1].props.children).toBe('Beta');
+    });
+
+    it('formats parent_company_os_id values as links', () => {
+        const value = ['US202511345DVTE', 'US202511345DFSD'];
+        const result = formatExtendedField({
+            ...baseProps,
+            value,
+            field_name: 'parent_company_os_id',
+        });
+
+        expect(result.primary).toHaveLength(2);
+        result.primary.forEach((element, idx) => {
+            expect(element.type).toBe('a');
+            expect(element.props.href).toBe(`/facilities/${value[idx]}`);
+            expect(element.props.children).toBe(value[idx]);
+        });
+    });
+
+    it('uses custom formatValue function', () => {
+        const result = formatExtendedField({
+            ...baseProps,
+            value: ['ABC'],
+            field_name: 'test_field',
+            formatValue: v => v.map(s => s.toLowerCase()),
```

Comment:
It is not recommended to use a single letter as a variable name. Could you please provide a more meaningful name instead of `v` and `s`?
Could you please review all the new code you’ve added?